src/controllers/bus.js

```diff
@@ -0,0 +1,93 @@
+import successRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+import { paginate } from 'paginate-info'
+import model from '../database/models';
+const {Bus}=model;
+export const createBus = async (req,res)=>{
+    try {
+        const bus= await Bus.create(req.body);
+        if(!bus){
+            return errorRes(res, 404, 'There was an error while creating a bus');
+        } else {
+            return successRes(res, 201, 'Bus created successfully', bus);
+        }
+        
+    } catch (error) {
+      return errorRes(res, 500,error);
+      
+    }
+}
+export const getBus = async (req, res) => {
+    try {
+      const { query: { page = 1, limit = 10 } } = req;
+      const offset = (page - 1) * limit;
+      const { rows, count } = await Bus.findAndCountAll({
+        page,
+        limit,
+        offset,
+      });
+      const pagination = paginate(page, count, rows, limit);
+  
+      if (offset >= count) {
+        res.status(401).json({
+          message: res.__('There are no buses registered in the system'),
+        });
+        errorRes(res,401,'There are no buses registered in the system')
```

Comment:
the error messages are not in match with the status codes. And what's the difference between line 32 and line 35?